lib/macho/macho_file.rb

```diff
@@ -420,13 +420,13 @@ def delete_rpath(path)
     # @return [Array<MachO::Section64>] if the Mach-O is 64-bit
     def sections(segment)
       section_klass = case segment
-      when MachO::SegmentCommand
-        MachO::Section
-      when MachO::SegmentCommand64
-        MachO::Section64
-      else
-        raise ArgumentError, "not a valid segment"
-      end
+                      when MachO::SegmentCommand
+                        MachO::Section
+                      when MachO::SegmentCommand64
+                        MachO::Section64
+                      else
+                        raise ArgumentError, "not a valid segment"
+                      end
```

Comment:
Yeah, I do generally prefer the original form.
I'll add those instructions to the `.rubocop.yml`.